tests/playwright-test/watch.spec.ts

```diff
@@ -89,3 +89,81 @@ test('should print dependencies in ESM mode', async ({ runInlineTest, nodeVersio
     'b.test.ts': ['helperA.ts', 'helperB.ts'],
   });
 });
+
+test('should perform initial run', async ({ runWatchTest }) => {
+  const testProcess = await runWatchTest({
+    'a.test.ts': `
+      pwt.test('passes', () => {});
+    `,
+  }, {});
+  await testProcess.waitForOutput('a.test.ts:5:11 › passes');
+  await testProcess.waitForOutput('Waiting for file changes.');
+});
+
+test('should quit on Q', async ({ runWatchTest }) => {
+  const testProcess = await runWatchTest({}, {});
+  await testProcess.waitForOutput('Waiting for file changes.');
+  testProcess.process.stdin!.write('q');
```

Comment:
Perhaps introduce a helper `TestChildProcess.write()`?

Reply:
I'll follow up if we get many